app/assets/javascripts/forms.js.coffee

```diff
@@ -1,24 +1,60 @@
 $ ->
+  class Selectizer
+    constructor: (el) ->
+      @$el = $(el)
```

Comment:
Do you actually need to carry around the dom element, or can you pull out the relevant attributes and keep the class light?